sql/updates/world/2016_03_19_00_world.sql

```diff
@@ -0,0 +1,3 @@
+-- Blinding Light
+DELETE FROM `spell_script_names` WHERE `spell_id` = 115750;
```

Comment:
I've been the only thing I do not understand why we remove the name script and not by id spells . After all, if there is a similar spell with another script that will fill the same mistake.